core/src/main/java/com/graphhopper/routing/QueryGraph.java

```diff
@@ -694,6 +702,9 @@ private boolean isInitialized() {
     }
 
     @Override
+    /**
+     * @see QueryGraph
```

Comment:
Sorry, I confused this method with `lookup`. The behavior of the edge explorer is also explained in the class comment:

>However, the edge explorer returned by {@link #createEdgeExplorer()} only returns two virtual edges per virtual node (the ones with correct base node).